code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -558,7 +558,11 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 		if(temp)
 			helptext = "<span class='warning'>Experimental gland detected!</span>"
 		else
-			helptext = "<span class='notice'>Subject suitable for experiments.</span>"
+			var/obj/item/organ/heart/hasheart = L.getorganslot(ORGAN_SLOT_HEART)
+			if (hasheart)
```

Comment:
could just do `if(L.getorganslot(ORGAN_SLOT_HEART)` instead of a new var, same goes to the `temp` over there, but that's not that big of a deal I guess